eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -97,80 +97,84 @@ object Helpers {
   /**
    * Called by the fundee
    */
-  def validateParamsFundee(nodeParams: NodeParams, features: Features, open: OpenChannel, remoteNodeId: PublicKey): Unit = {
+  def validateParamsFundee(nodeParams: NodeParams, features: Features, open: OpenChannel, remoteNodeId: PublicKey): Option[ChannelException] = {
```

Comment:
For consistency, and to better convey the idea that we expect a `Right`?

```suggestion
  def validateParamsFundee(nodeParams: NodeParams, features: Features, open: OpenChannel, remoteNodeId: PublicKey): Left[ChannelException, Done] = {
```

NB: this is the builtin `akka.Done`

Reply:
Yes, that was on my todo list when getting back to this PR, it should be an Either[ChannelException, Done]` or `Either[ChannelException, Unit]`. Let me fix that!

Reply:
Done in https://github.com/ACINQ/eclair/pull/1585/commits/737b51da59fa27b795ba2edeb27ce4fc47efc1c8